<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>481</width>
    <height>611</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="toolTipDuration">
       <number>-6</number>
      </property>
      <widget class="QWidget" name="pageMissingCategory">
       <widget class="QListWidget" name="QlistCategories">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>120</y>
          <width>181</width>
          <height>411</height>
         </rect>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
       </widget>
       <widget class="QPushButton" name="QbuttonSkip">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>180</y>
          <width>141</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Set category but
 don't add permenantly</string>
        </property>
       </widget>
       <widget class="QPushButton" name="QbuttonSelect">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>120</y>
          <width>141</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>Set category permenantly</string>
        </property>
       </widget>
       <widget class="QTextEdit" name="QtextMissing_ingredient">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>60</y>
          <width>181</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="QlabelMissing_ingredient">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>40</y>
          <width>171</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Ingredient without category:</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="QlabelCategories">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>100</y>
          <width>241</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Select category for missing ingredient from list</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="QlabelMissing_ingredient_title">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>321</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Missing Category for Grocery Item</string>
        </property>
       </widget>
       <widget class="QLabel" name="QlastAction">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>240</y>
          <width>131</width>
          <height>71</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="pageAddRecipe"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>481</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuList_management">
    <property name="title">
     <string>List management</string>
    </property>
    <addaction name="QActionSort_grocery_list"/>
    <addaction name="actionAdd_recipe_to_list"/>
   </widget>
   <addaction name="menuList_management"/>
  </widget>
  <action name="QActionSort_grocery_list">
   <property name="text">
    <string>Sort grocery list</string>
   </property>
  </action>
  <action name="actionAdd_recipe_to_list">
   <property name="text">
    <string>Add recipe to list</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
